File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.py
num=[0]*31
absn=[0]*2
idx=0
for i in range(28):
    num[int(input())]=1
for i in range(1,31):
    if num[i]==0:
        absn[idx]=i
        idx+=1
for i in range(2):
    print(absn[i])

File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.c
#include <stdio.h>

int main(void) {
    int elem,idx=0,num[31]={0,},absn[2];
    for(int i=0;i<28;i++) {
        scanf("%d",&elem);
        num[elem]=1;
    }
    for(int i=1;i<31;i++) {
        if(num[i]==0)
            absn[idx++]=i;
    }
    for(int i=0;i<2;i++)
        printf("%d\n",absn[i]);
    return 0;
}